Resources/MixedModels/qml/MixedModelsBGLMM.qml

```diff
@@ -307,15 +315,16 @@ Form {
 				}
 			}
 
-			CIField
-			{
-				name:	"summaryCI"
-				label:	"Confidence interval"
-			}
+		}
 
-			SetSeed{}
+		SetSeed{}
 
+		CIField
+		{
+			name:	"summaryCI"
+			label:	"Confidence interval"
```

Comment:
Should not this be "Credible interval"? (Assuming "BGLMM" means Bayesian...)

Also, you forgot `qsTr()` here

Reply:
Yes, you are right. I will change this in a separate PR once https://github.com/jasp-stats/jasp-desktop/pull/4167 is merged (to avoid conflicts)